frontends/election-manager/src/screens/logic_and_accuracy_screen.tsx

```diff
@@ -13,25 +13,43 @@ export function LogicAndAccuracyScreen(): JSX.Element {
   return (
     <NavigationScreen>
       <Prose maxWidth={false}>
-        <h1>L&amp;A Materials</h1>
+        <h1>L&A Testing Documents</h1>
         {isLiveMode ? (
           <p>
-            L&amp;A materials are not available after live CVRs have been
-            imported.
+            L&A materials are not available after live CVRs have been imported.
```

Comment:
Should this read "L&A testing documents are not available..." for consistency with the "L&A Testing Documents" title? Don't feel strongly about this!

Reply:
Good catch. Changing to "L&A testing documents are not available after official election CVRs have been imported."